pkg/services/vmoperator/vmopmachine.go

```diff
@@ -188,6 +193,116 @@ func (v *VmopMachineService) ReconcileNormal(ctx context.Context, machineCtx cap
 	// Set the VM state. Will get reset throughout the reconcile
 	supervisorMachineCtx.VSphereMachine.Status.VMStatus = vmwarev1.VirtualMachineStatePending
 
+	var affInfo affinityInfo
+	if feature.Gates.Enabled(feature.NodeAutoPlacement) &&
+		!infrautilv1.IsControlPlaneMachine(machineCtx.GetVSphereMachine()) {
+		vmOperatorVMGroup := &vmoprv1.VirtualMachineGroup{}
```

Comment:
may be
```suggestion
		vmGroup := &vmoprv1.VirtualMachineGroup{}
```
(it goes without saying it is from VM operator, so let's get rid of it to improve readability)